recipes/ligo-camera-client/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "ligo-camera-client" %}
+{% set version = "2.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/cds/software/cameras/camera-client/-/archive/{{ version }}/camera-client-{{ version }}.tar.gz
+  sha256: 854ec7eea637bfc89622068d123cb4faf239cd5864256b4364054f0be84ee976
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('gxx') }}
```

I think this should be

```suggestion
    - {{ compiler('cxx') }}
```